lib/SMTP/SmtpClientFactory.php

```diff
@@ -76,7 +76,7 @@ public function create(Account $account): Horde_Mail_Transport {
 			'port' => $mailAccount->getOutboundPort(),
 			'username' => $mailAccount->getOutboundUser(),
 			'secure' => $security === 'none' ? false : $security,
-			'timeout' => (int)$this->config->getSystemValue('app.mail.smtp.timeout', 2)
+			'timeout' => (int)$this->config->getSystemValue('app.mail.smtp.timeout', 10)
```

Comment:
This is a known problem on some setups, hence this is configurable via `config/config.php` (the `app.mail.smtp.timeout` key).

I suggest to simply change the value on your setup. The default timeout is fine IMO. cc @nextcloud/mail

Reply:
True, however the error message does not indicate the cause. The [IMAP timeout](https://github.com/nextcloud/mail/blob/master/lib/Account.php#L145) is 20 seconds.

Reply:
Good point. I think the default IMAP timeout should be lower :wink:

Reply:
Yea that makes sense. Should I change this PR to lower the IMAP timeout instead?

Reply:
Yes, please. How about we make both 5s by default for both? Sounds like a fair compromise to me.